styles/layout.css

```diff
@@ -0,0 +1,32 @@
+section {
```

Comment:
Update consistency of spacing between css calls. Choose one or the other.

{
/* Comment*/
.class

Or 

{
/* Comment*/
.class